playbooks/inventory.txt

```diff
@@ -0,0 +1,6 @@
+[node]
+54.237.64.167 ansible_ssh_private_key_file=~/.ssh/groupone-awskey.pem
```

Comment:
Can you make sure everyone has the key?

Reply:
I've notified everyone in the group to add the key into their ssh file